### PR TITLE
Splits Identity into its own specification.

### LINKED ISSUE #1914: Split Identity into its own specification
As proposed on the mailing list on May 23 2018, and per consensus reached in the June 2018 F2F meeting.

### PR DESCRIPTION
**Pull request by alvestrand**
This takes care of some of the linkages.
Fixes #1914

This keeps the file in the webrtc-pc repo. Moving it into its
own repo can be done administratively.

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/webrtc-pc/pull/1915.html" title="Last updated on Jul 11, 2018, 9:23 AM GMT (bcdca93)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/webrtc-pc/1915/8938421...bcdca93.html" title="Last updated on Jul 11, 2018, 9:23 AM GMT (bcdca93)">Diff</a>

### CODE REVIEW

**Comment by alvestrand** [comment]
Note - still a couple of link errors, and we need a bibliography entry to complete the linkage. Asking @dontcallmedom to take care of that.

**Comment by alvestrand** [comment]
The Travis failure is because [[IDENTITY]] is not a defined reference.

**Comment by aboba** [comment]
Prior to splitting out Identity, I would suggest merging PRs https://github.com/w3c/webrtc-pc/pull/1899, https://github.com/w3c/webrtc-pc/pull/1900 and https://github.com/w3c/webrtc-pc/pull/1901

File: identity.html [hunk 1]
```diff
@@ -0,0 +1,971 @@
...
+<head>
+  <meta charset="utf-8">
+  <link href="webrtc.css" rel="stylesheet">
+  <title>Identity for WebRTC 1.0</title>
+  <script class="remove" src="respec-w3c-common.js" type="text/javascript">
+  // // keep this comment //
+  </script>
+  <script class="remove" src="identity.js" type="text/javascript">
+  // // keep
+    this comment //
```
Comment (martinthomson) [inline identity.html]: Problem?

**Comment by martinthomson** [comment]
You probably want [[!WEBRTC-IDENTITY]].  And I would prefer that we clear PRs first also.

What was the conclusion with isolated streams?  It seems like they don't really make sense without identity.

File: identity.html (hunk 1 above)
Comment (alvestrand) [reply identity.html]: Copied verbatim from the existing draft. I was afraid there might be magic in the linebreak.

**Comment by alvestrand** [comment]
I pushed a version that was rebased on the post-patch-thursday webrtc, with 2 copies of the body of the identity spec. This is expected to be messy, but should prevent me from forgetting the thursday patches.

**Comment by alvestrand** [comment]
WRT isolated streams, I think we should file a separate bug on moving that to identity. Filed #1929.

This should now be ready for review. I think I have all the links fixed.

**Pull request merged**